sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -489,5 +524,16 @@ private static BlobTriggerMessage GetBlobTriggerMessage(JObject jo, string funct
                 FunctionId = functionId
             };
         }
+
+        /// <summary>
+        /// Type used to provide blob information that can be passed to
+        /// ParameterBindingData to enable workers to create a blob client
```

Comment:
nit: remove any AzureFunctions specific stuff like "workers" and keep this general. Feel free to remove the comment completely since it's a private type in this class.